diff --git a/src/Driver/Cores/CoreInterface.php b/src/Driver/Cores/CoreInterface.php
index ff23034..929a714 100644
--- a/src/Driver/Cores/CoreInterface.php
+++ b/src/Driver/Cores/CoreInterface.php
@@ -97,13 +97,15 @@ interface CoreInterface extends OriginalCoreInterface {
   /**
    * Deletes an entity permanently.
    *
+   * @param string $entity_type
+   *   Entity type identifier.
    * @param object $entity
    *   The drupal entity, appropriately typed.
    *
    * @throws \Exception
    *   In case of failures an exception is thrown.
    */
-  public function entityDelete($entity);
+  public function entityDelete($entity_type, $entity);
 
   /**
    * Add a translation for an entity.
diff --git a/src/Driver/Cores/Drupal8.php b/src/Driver/Cores/Drupal8.php
index abba854..2b91fc0 100644
--- a/src/Driver/Cores/Drupal8.php
+++ b/src/Driver/Cores/Drupal8.php
@@ -266,7 +266,7 @@ class Drupal8 extends OriginalDrupal8 implements CoreInterface {
   /**
    * {@inheritdoc}
    */
-  public function entityDelete($entity) {
+  public function entityDelete($entity_type, $entity) {
     $entity->delete();
   }
 
